spec/models/recruitment_cycle_spec.rb

```diff
@@ -13,6 +13,10 @@
 require "rails_helper"
 
 describe RecruitmentCycle, type: :model do
+
+  let(:current_cycle) { TestSetup.current_cycle }
+  let(:next_cycle) { TestSetup.next_cycle }
+
   subject { find_or_create :recruitment_cycle, year: "2019" }
```

Comment:
Would it make sense, as part of this PR, to change this too? We would eventually want to do that, and since we're changing all the other cycles in this file already, maybe it would (make sense).